content/en/docs/how_to/enable_prometheus_metrics.md

```diff
@@ -0,0 +1,95 @@
+
+```text
+# HELP http_request_duration_seconds duration histogram of http responses labeled with: status_code
+# TYPE http_request_duration_seconds histogram
+http_request_duration_seconds_bucket{le="0.003",status_code="200"} 0
+http_request_duration_seconds_bucket{le="0.03",status_code="200"} 0
+http_request_duration_seconds_bucket{le="0.1",status_code="200"} 1
+http_request_duration_seconds_bucket{le="0.3",status_code="200"} 1
+http_request_duration_seconds_bucket{le="1.5",status_code="200"} 1
+http_request_duration_seconds_bucket{le="10",status_code="200"} 1
+http_request_duration_seconds_bucket{le="+Inf",status_code="200"} 1
+http_request_duration_seconds_sum{status_code="200"} 0.053406219
+http_request_duration_seconds_count{status_code="200"} 1
+
+# HELP up 1 = up, 0 = not up
+# TYPE up gauge
+up 1
+```
+
+## Configuration
+
+Installation will install a default configuration file, `conf/prometheus.default.js` and the next time your application starts, metrics can be accessed at: http://localhost:8081/metrics. The `/metrics` endpoint as well as the port the metrics are bound are configurable via the configuration file. By default, the plugin will bind to `8081`, but this can be changed in the configuration or by setting the environment variable `API_BUILDER_PROMETHEUS_PORT`. The metrics do not persist between application restarts.
+
+```js
+module.exports = {
+  pluginConfig: {
+    '@axway/api-builder-plugin-prometheus': {
+      enabled: true,
+      http: {
+        // note that the port must be different than the runtime
+        port: process.env.API_BUILDER_PROMETHEUS_PORT || 8081
+      },
+      options: {
+        // These are the main options and their defaults. For the
+        // complete list of options, see:
+        // https://www.npmjs.com/package/express-prom-bundle#options
+        metricsPath: '/metrics',
+        includeStatusCode: true,
+        includeMethod: false,
+        includePath: false,
+        includeUp: true,
+        metricType: 'histogram'
+      }
+    }
+  }
+};
+```
+
+### Options
+
+* **enabled** (`boolean`): Enables or disables the plugin. Default: `true`.
+* **http.port** (`integer|string`): Sets the metrics port (note that `process.env.API_BUILDER_PROMETHEUS_PORT` takes precedence). The port **cannot** be the same port that {{% variables/apibuilder_prod_name %}} is using. Default `8081`.
+* **options** (`object`): These are the `express-prom-bundle` options, such as the route used to access the metrics (defaults to `/metrics`). See [options](https://www.npmjs.com/package/express-prom-bundle#options) for further details.
+
+### Docker
+
+If you are running {{% variables/apibuilder_prod_name %}} from Docker, then you will want to expose the port {{% variables/apibuilder_prod_name %}} is running on (e.g. 8080), as well as the Prometheus metrics (e.g. 8081).
+
+```bash
+docker build -t myapp ./
+docker run --name myapp -e PORT=8080 -e API_BUILDER_PROMETHEUS_PORT=8081 myapp:latest
```

Comment:
I don't think the ports will actually be exposed here. only set.
`docker run --name myapp -e PORT=8080 -e API_BUILDER_PROMETHEUS_PORT=8081 -p 8080:8080 -p 8081:8081 myapp:latest`

Reply:
they worked for me when I tested it.  I believe the colon will map it, e.g. 8080:80

Reply:
I tested with these commands and it didn't expose the ports :(